readme.md

```diff
@@ -5,28 +5,29 @@
 
 # Microsoft Azure IoT SDKs
 
-This repository references Microsoft Azure IoT device SDKs and IoT service SDKs. Device SDKs enable you connect client devices to Azure IoT Hub. Service SDKs enable you to manage your IoT Hub service instance.
+This repository serves as a landing page for all Azure IoT SDKs, each being maintained in their own repository, including sample code and documentation.  Visit [this page](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks) to learn more about Azure IoT SDKs.
 
 Visit http://azure.com/iotdev to learn more about developing applications for Azure IoT.
 
 ## Microsoft Azure IoT SDKs
 
-The Microsoft Azure IoT **Device** SDKs contain code that facilitate building devices and applications that connect to and are managed by Azure IoT Hub services.
-The Microsoft Azure IoT **Service** SDKs help you to build applications that interact with your devices and manage device identities in your IoT hub.
+The Microsoft Azure IoT **Device** SDKs:
+- [Azure IoT Device SDK for C](https://github.com/Azure/azure-iot-sdk-c) written in ANSI C (C99) for portability and broad platform compatibility. There are two device client libraries for C, the low-level iothub_client and the serializer.
+- [Azure IoT Device SDK for Python](https://github.com/Azure/azure-iot-sdk-python/tree/master/device)
+- [Azure IoT Device SDK for Node.js](https://github.com/Azure/azure-iot-sdk-node/tree/master/device)
+- [Azure IoT Device SDK for Java](https://github.com/Azure/azure-iot-sdk-java/tree/master/device)
+- [Azure IoT Device SDK for .NET](https://github.com/Azure/azure-iot-sdk-csharp/tree/master/device)
 
-Devices and data sources in an IoT solution can range from a simple network-connected sensor to a powerful, standalone computing device. Devices may have limited processing capability, memory, communication bandwidth, and communication protocol support. The IoT device SDKs enable you to implement client applications for a wide variety of devices.
-
-This repository serves as a landing page for all SDKs, each being actually maintained in their own repository, including sample code and documentation:
-
-- [Azure IoT SDK for C](https://github.com/azure/azure-iot-sdk-c)
-- [Azure IoT SDK for Python](https://github.com/azure/azure-iot-sdk-python)
-- [Azure IoT SDK for Node.js](https://github.com/azure/azure-iot-sdk-node)
-- [Azure IoT SDK for Java](https://github.com/azure/azure-iot-sdk-java)
-- [Azure IoT SDK for .NET](https://github.com/azure/azure-iot-sdk-csharp)
+The Microsoft Azure IoT **Service** SDKs:
+- [Azure IoT Service SDK for C](https://github.com/Azure/azure-iot-sdk-c/tree/master/iothub_service_client) written in ANSI C (C99) for portability and broad platform compatibility. There are two device client libraries for C, the low-level iothub_client and the serializer.
```

Comment:
This part needs to be changed: "There are two device client libraries for C, the low-level iothub_client and the serializer. "
It is a C&P from above.

It should be "There are two service client libraries for C, the low-level iothub_messaging_ll and iothub_messaging."

Reply:
Definitely, will update this